czech/kapsy/nuz/bandaz/dvere/dvere.md

```diff
@@ -0,0 +1,3 @@
+Projdeš dveřmi azjistiš že je tam hromada maršmeloidních humanoidu kteří vypadají jako panaček mischelin obraceny naruby 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.